src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -145,6 +145,11 @@ public MesosCloud(
         this.master);
   }
 
+  @Override
```

Comment:
you can just comment this out with a TODO and merge the PR whenever doesn't need to be dependent on the core PR

Reply:
Commented out with a TODO in 73d45ab1d7e3c03ab931a02033b4f7415734e14c